metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorDatabaseService.java

```diff
@@ -188,6 +189,10 @@ public boolean exists(@Nonnull @NonNull final ConnectorContext requestContext,
                 }
                 qualifiedNames.add(qualifiedName);
             }
+            //supporting sort by qualified name only
+            if (sort != null) {
+                ConnectorUtils.sort(qualifiedNames, sort, Comparator.comparing(QualifiedName::toString));
```

Comment:
Sure, I use the whole qualified name for sorting, even though the catalog name should be the same for now.